Formula/llvm.rb

```diff
@@ -211,7 +227,6 @@ def install
 
       if OS.mac?
         extra_args << "-DLLVM_ENABLE_LIBCXX=ON"
-        extra_args << "-DDEFAULT_SYSROOT=#{macos_sdk}" if macos_sdk
```

Comment:
Seems like a notable change this. What's changed?

Reply:
This is for the extra compilers built during PGO that are never installed. CMake manages those compilers (and sets the sysroot for them c/o `CMAKE_OSX_SYSROOT`), so there's no reason to set a default sysroot for them. The default sysroot is also behind some of the errors that you see `rescue`s for.

To be clear, we still set `DEFAULT_SYSROOT` for the compiler that is eventually installed.

Reply:
Ok makes sense 👍